src/main/java/com/bazel-diff/main.java

```diff
@@ -80,13 +80,19 @@ public Integer call() {
     @Option(names = {"-m", "--modifiedFilepaths"}, description = "The path to a file containing the list of modified filepaths in the workspace, you can use the 'modified-filepaths' command to get this list")
     File modifiedFilepaths;
 
+    @Option(names = {"-so", "--bazelStartupOptions"}, description = "Additional Bazel client startup options used when invoking Bazel")
+    String bazelStartupOptions;
+
+    @Option(names = {"-co", "--bazelCommandOptions"}, description = "Additional Bazel command options used when invoking Bazel")
+    String bazelCommandOptions;
+
```

Comment:
These should not be needed, you can refer to these values off of parent like so

`parent.bazelCommandOptions `

Reply:
@tinder-maxwellelliott Fixed it